lib/policies/proxy/proxy.js

```diff
@@ -77,7 +77,10 @@ module.exports = function (params, config) {
 
   const balancer = createStrategy(strategy, proxyOptions, urls);
 
-  const stripPathFn = params.stripPath ? (req) => { req.url = `${req.params[0]}${req._parsedUrl.search || ''}`; } : () => { };
+  const stripPathFn = params.stripPath
+    ? req => {
+      req.url = `${req.params[0] || ''}${req._parsedUrl.search || ''}`;
```

Comment:
`req.params[0] || ''`

Should this be `req.params[0] || '/'`?  I see that the test is passing, so `/` might be assumed somewhere, but I think `req.url = ''`--which is what this can appear to be--is confusing.

Reply:
Tests pass with both things (`''` and `/`). I definitely recall there's some normalization somewhere but I can't remember where unfortunately. It should be good.